### ISSUE TITLE
[Bug] Selected Meter Aidon and Kamstrup

### ISSUE THREAD

**Issue by xibriz** [roarfred/AmsToMqttBridge#41]
When booting as AP an selecting meter, the numbers are:

1: kaifa
2: Aidon
3: Kamstrup

But in the `switch (ap.config.meterType)` Aidon is 3 and Kamstrup is 2.

Also, I don't see where you set the hanreader to SERIAL_8N1 for Kamstrup.

**Comment by roarfred** [comment]
Good catch, thanks! Both are valid issues that will have to be corrected
